moderation/reports/customReport.go.tmpl

```diff
@@ -0,0 +1,87 @@
+{{/*
+    This handy-dandy custom command-bundle allows a user to cancel their most recent report and utilizes
+    Reactions to make things easier for staff.
+    This custom command is basically the native report-command, but adds some back-end functionalites in order for the rest to work :)
+
+    Usage: `-ru <User:Mention/ID> <Reason:Text>`
+
+    Recommended Trigger type and trigger: Regex trigger with trigger `\A-r(eport)?(?:u(ser)?)?(\s+|\z)`
+
+    Created by: Olde7325 <https://github.com/Olde7325>
+*/}}
+
+{{/*CONFIG AREA START*/}}
+
+{{$REPORT_LOG := 772251753173221386}} {{/*The Channel-ID of your report logging channel.*/}}
+{{$REPORT_DISCUSSION := 766370841196888104}} {{/*The Channel-ID of the channel were users should talk to staff.*/}}
+
+{{/*CONFIG AREA END*/}}
+
+
+{{/*ACTUAL CODE*/}}
+{{$p := index (reFindAllSubmatches `.*?: \x60(.*)\x60\z` (execAdmin "prefix")) 0 1}}
+{{$Escaped_Prefix := reReplace `[\.\[\]\-\?\!\\\*\{\}\(\)\|]` $p `\${0}`}}
+{{if not (reFind (print `\A` $Escaped_Prefix `|<@!?204255221017214977>`) .Message.Content)}}
+{{$response:= sendMessageRetID nil "Did not set regex to match Server Prefix!"}}{{deleteTrigger}}
+{{deleteMessage nil $response}}{{else}}
+{{if and .CmdArgs (lt (len .CmdArgs) 2)}}
+    {{if eq (index .CmdArgs 0) "dbSetup"}}
+        {{if (in (split (index (split (exec "viewperms") "\n") 2) ", ") "ManageServer")}}
+            {{dbSet 2000 "reportSettings" (sdict "reportLog" (str $REPORT_LOG
```

Comment:
Saving `sdict` to database preserves the original type. So you can consider not stringifying the IDs and using them directly.. just a thought.